addons/kube-state-metrics/parameter.cue

```diff
@@ -0,0 +1,24 @@
+parameter: {
+
+    // global parameters
+
+    // +usage=The namespace of the kube-state-metrics to be installed
+    namespace: *"o11y-system" | string
+    // +usage=The name of the addon application
+    name: *"addon-kube-state-metrics" | string
```

Comment:
Support for name customization is not recommended.

Reply:
OK